qcfractal/queue/fireworks_adapter.py

```diff
@@ -4,19 +4,32 @@
 
 import logging
 
-import fireworks
-import fireworks.core.rocket_launcher
+try:
+    import fireworks
+    import fireworks.core.rocket_launcher
+except ImportError:
+    fireworks = None
 
 
 class FireworksAdapter:
```

Comment:
Probably, but at the moment there isn't a benefit to the ABC's as they are currently fully implemented. This will change as we acquire more adapters. Something to revisit.